### PR TITLE
feat: upgrading to PyPI reinvent models 0.0.14 version

### PR DESCRIPTION
**Pull request by Ashish13898**
This PR contains all the changes related to upgrading the package to the latest version of reinvent models in PyPI ( 0.0.14 ).

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -6,8 +6,8 @@
 setuptools.setup(
     name="reinvent_models",
     version="0.0.7",
```
Comment (drugilsberg) [inline setup.py]: is this version correct?

File: setup.py [hunk 2]
```diff
@@ -6,8 +6,8 @@
 setuptools.setup(
     name="reinvent_models",
     version="0.0.7",
-    author="PaccMann Team",
-    description="Generative models for Reinvent adapted for PaccMann",
+    author="GT4SD Team",
+    description="Generative models for Reinvent adapted for GT4SD",
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/PaccMann/reinvent_models",
```
Comment (drugilsberg) [inline setup.py]: fix link

**Review by drugilsberg — COMMENTED** [review]
Looks good, but I would make sure we do not apply typing and styling to make more straightforward an eventual contribution to Reinvent.

File: setup.py (hunk 1 above)
Comment (Ashish13898) [reply setup.py]: I have just pushed an update for this :)

File: setup.py (hunk 2 above)
Comment (Ashish13898) [reply setup.py]: I have just pushed an update for this :)

**Comment by Ashish13898** [comment]
> Looks good, but I would make sure we do not apply typing and styling to make more straightforward an eventual contribution to Reinvent.

Yes, I have basically replaced the whole package with the latest version and applied the changes we did last time.

**Review by drugilsberg — APPROVED** [review]
Looks great @Ashish13898

**Pull request merged**